toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -603,6 +651,27 @@ func (g *PkgGraph) AllNodesFrom(rootNode *PkgNode) []*PkgNode {
 	return nodes
 }
 
+func (g *PkgGraph) PrintLookupTable() {
```

Comment:
This looks like it's only used internally. Please make it a private function. Especially since the user of the graph doesn't need to know about the lookup table.
```suggestion
func (g *PkgGraph) printLookupTable() {
```